### ISSUE TITLE
The question about training

### ISSUE THREAD

**Issue by 201700301267** [yizhiwang96/deepvecfont-v2#4]
I trained the model using the Chinese dataset you provided, but there were issues during the training process. The values of Loss function all become ‘nan'. And the following warning appeared: “WARNING:root:NaN or Inf found in input tensor”.
Can you explain the reason for the problem？

**Comment by yizhiwang96** [comment]
> I trained the model using the Chinese dataset you provided, but there were issues during the training process. The values of Loss function all become ‘nan'. And the following warning appeared: “WARNING:root:NaN or Inf found in input tensor”. Can you explain the reason for the problem？

Hi, Thank you for your feedback. I am trying to re-run the training process, and see if the problem will occur to me and try to spot the problem.

**Comment by 201700301267** [comment]
I set the batch_size to 8 and 16 during the training process, but did not follow the numbers you provided. Is the error related to this？

**Comment by yizhiwang96** [comment]
> I set the batch_size to 8 and 16 during the training process, but did not follow the numbers you provided. Is the error related to this？

Were you using the environment I suggested? I ran this experiment again and didn't get Nan loss.

**Comment by 201700301267** [comment]
I used the same conda environment as deepvecfont-v1，the python version is also 3.9, will this affect the training process？

**Comment by yizhiwang96** [comment]
> I used the same conda environment as deepvecfont-v1，the python version is also 3.9, will this affect the training process？

Sometimes a different version of pytorch could lead to NaN. Could you try the enviroment I provided in this repo?
